modules/glfw/src/glfwmodule.cpp

```diff
@@ -75,19 +81,23 @@ void GLFWModule::onProcessorNetworkEvaluationEnd() {
     // its work before we continue. This is needed to make sure that we have textures that are upto
     // data when we render the canvases.
     auto syncObj = glFenceSync(GL_SYNC_GPU_COMMANDS_COMPLETE, 0);
```

Comment:
Is this not callled when the exception is thrown? Should check if it exist in that case

Reply:
how do you mean? what exception? you mean if a processor fail in the network?

Reply:
this will always be call even if a processor fail, but that should not be a problem

Reply:
There is an exception thrown in the constructor if not glFenceSync exist. Then it is used here

Reply:
if the constructor throws then the destructor is never called